ui/src/components/ResultsTable/ResultsTableBody.js

```diff
@@ -14,7 +14,7 @@ export default class ResultsTableBody extends Component {
 
   renderRow = (columns, rows) => {
     return rows.map((row, rowIndex) => {
-      const identifier = row.pid ? row.pid : row.id;
+      const identifier = row.pid ? row.pid : row.id ? row.id : row.pid_value;
```

Comment:
Maybe we can address this in a serializer level so we don't see these comparisons around.

Reply:
yes! We will, I didn't want to start changing this too... I agree with you